octoprint_octorelay/const.py

```diff
@@ -5,13 +5,14 @@
 STARTUP = "STARTUP"
 PRINTING_STARTED = "PRINTING_STARTED"
 PRINTING_STOPPED = "PRINTING_STOPPED"
+TURNED_ON = "TURNED_ON"
 
 # Task cancellation exceptions
 # { eventHappened: [ events which postponed timers should NOT be cancelled ]
 CANCELLATION_EXCEPTIONS = {}
 
 # Versioning of the plugin's default settings described below
-SETTINGS_VERSION = 3
+SETTINGS_VERSION = 4
```

Comment:
```suggestion
SETTINGS_VERSION = 3
```

I'm not sure if migration is needed in this case